make/common/ZipArchive.gmk

```diff
@@ -26,6 +26,8 @@
 ifndef _ZIP_ARCHIVE_GMK
 _ZIP_ARCHIVE_GMK := 1
 
+include ../ToolsJdk.gmk
```

Comment:
Should probably add a comment about inclusion of this file now requires an explicit dependency on build-tools in Main.gmk.

Reply:
done